tests/e2e/run.sh

```diff
@@ -34,8 +34,8 @@ test_e2e() {
     info "Creating mocked compliance operator data for compliance v1 tests"
     "$ROOT/tests/complianceoperator/create.sh"
 
```

Comment:
If deploy_stackrox is run before deploy_optional_e2e_components sensor will have an unnecessary restart, add a comment here explaining this